fakebook/src/app/service/upload-image.service.ts

```diff
@@ -19,7 +19,7 @@ export class UploadImageService {
     };
 
     return this.http.post<{ path: string, userId: number }>(
-      `${this.baseUrl}/api/ProfilePicture`,
+      `${environment.baseUrl}/api/ProfilePicture`,
```

Comment:
if this is a route we're using, it should be added to kubernetes_ingresses.yaml.

Reply:
added to k8